README.md

```diff
@@ -26,6 +26,17 @@ conditions. See the [LICENSE] file for details.
 
 1. Visual Studio Code with "Remote Development" installed
 
+### Sharing Git credentials with your container
```

Comment:
I added this as a subsection rather than as part of the numbered list to

1. Suggest that it's not a hard requirement (you _can_ get devcontainers to work without this)
2. Have more space to explain the nuances than just 1 bullet point

Reply:
Ah, in that case make it a separate how to and mention that here with appropriate links.  READMEs should stay small and skimmable and this is too much information for the README.

Reply:
Done, wdyt?